### PR TITLE
Feature/kafka simple extractor

### PR DESCRIPTION
**Pull request by akshayn1107**
Adds a Kafka Extractor for just pulling raw bytes from Kafka. This is useful if strings are being read off the Kafka log, or if no data processing is required through Gobblin.

### CODE REVIEW

File: gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaSimpleSource.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+/**
+ * A {@link KafkaSource} implementation for SimpleKafkaExtractor.
+ *
+ * @author akshay@nerdwallet.com
+ */
+public class KafkaSimpleSource extends KafkaSource<String, byte[]> {
+  /**
+   * Get an {@link Extractor} based on a given {@link WorkUnitState}.
+   * <p/>
```
Comment (zliu41) [inline gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaSimpleSource.java]: This line unnecessary

File: gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaSimpleExtractor.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.source.extractor.extract.kafka;
+
+import gobblin.configuration.WorkUnitState;
+import kafka.message.MessageAndOffset;
```
Comment (zliu41) [inline gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaSimpleExtractor.java]: Please add empty line before this line and use alphabetic ordering of imports.

**Comment by zliu41** [comment]
LGTM other than two minor comments.

**Comment by zliu41** [comment]
Also, there are too many commits in this PR, please merge them.

**Comment by akshayn1107** [comment]
@zliu41 I've squashed all the commits into one with rebase. Is that okay?

**Comment by zliu41** [comment]
@akshayn1107 Yes that'll work. Thanks.

**Pull request merged**